idl/grapefruit-seq.idol

```diff
@@ -0,0 +1,11 @@
+// Grapefruit Sequencer API
+Interface(
+    name: "Sequencer",
+    ops: {
+        "foo": (
+            args: {},
+            reply: Simple("u8"),
+            idempotent: true,
+        ),
```

Comment:
assuming this is a work in progress?

Reply:
Yeah, this is a placeholder to integrate the Idol compile + runtime stuff